app/src/chat_engine.py

```diff
@@ -50,9 +61,22 @@ def create_engine(engine_id: str) -> ChatEngineInterface | None:
 class GuruBaseEngine(ChatEngineInterface):
     datasets: list[str] = []
 
+    # Thresholds that determine which documents are sent to the LLM
+    retrieval_k: int = 8
```

Comment:
How was the threshold determined?

Reply:
Arbitrarily chosen: some value greater than the original 5 so that we send more docs to the LLM. Note that this is the default value and the user can adjust it.